actors/builtin/miner/miner_state_test.go

```diff
@@ -881,6 +881,23 @@ func TestMinerEligibleForElection(t *testing.T) {
 		eligible, err := miner.MinerEligibleForElection(store, actor.receiver, mSt, pSt, thisEpochReward, currEpoch)
 		require.NoError(t, err)
 		assert.False(t, eligible)
+
+		// get minimum
+		pow32GiBMin, err := abi.RegisteredSealProof_StackedDrg32GiBV1.ConsensusMinerMinPower()
+		require.NoError(t, err)
+
+		// with enough miners above minimum, power must be above min
+		pow := big.Sub(pow32GiBMin, big.NewInt(1))
+		pSt = constructPowerStateWithPowerAtAddr(t, pow, actor.receiver, abi.RegisteredSealProof_StackedDrg32GiBV1, rt)
+		pSt.MinerAboveMinPowerCount = power.ConsensusMinerMinMiners
```

Comment:
Looks like you're missing some assertions after here, because this is then overwritten.